deploy/FlapperInit.sol

```diff
@@ -143,22 +150,41 @@ library FlapperInit {
         dss.chainlog.setAddress(clKey, wrapper_);
     }
 
+    function initFarm(
+        DssInstance memory dss,
+        address            farm_,
+        FarmConfig  memory cfg
+    ) internal {
+        FarmLike     farm     = FarmLike(farm_);
+        SplitterLike splitter = SplitterLike(cfg.splitter);
+
+        require(farm.rewardsToken() == DaiJoinLike(cfg.daiJoin).dai(), "Farm rewards not dai");
+
+        require(cfg.hop >= 5 minutes, "hop too low");
+        require(cfg.hop == splitter.hop(), "hop mismatch");
```

Comment:
Not really assumes, it enforces it, otherwise the splitter hop would be 0, that's why added the >= 5 minutes as well, to avoid having 0 == 0 due to the splitter not being init before. But yeah adding a comment doesn't hurt.

Reply:
Ah ok, l didn't see that 5 minutes check as didn't pull today.
Let's still add a comment as you said.

Reply:
Comment added.